### ISSUE TITLE
MS SQL Scan with instance

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuza55/opendlp#46]
```
Hello,
I am trying to scan MS SQL database on a specific instance.

But i don't know where it is possible to specify the instance name in the 
profile or when i start a new scan.

Is it only possible to scan the deault instance in SQL?

Thank you for your help.

Regards,
Fabrice


```

Original issue reported on code.google.com by `fabrice....@gmail.com` on 16 Feb 2012 at 4:20

**Comment by GoogleCodeExporter** [comment]
```
Currently it is not possible. I will look at adding this in a future release, 
perhaps even the next one. Thanks for the heads up.
```

Original comment by `andrew.O...@gmail.com` on 16 Feb 2012 at 7:56
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for your reply.

Do you have an idea for next release date approx.?

Regards,
Fabrice
```

Original comment by `fabrice....@gmail.com` on 17 Feb 2012 at 10:13

**Comment by GoogleCodeExporter** [comment]
```
It depends on my workload, but hopefully within the next few weeks (if not 
sooner).

On what version of MSSQL are you trying OpenDLP, by the way?
```

Original comment by `andrew.O...@gmail.com` on 18 Feb 2012 at 3:28

**Comment by GoogleCodeExporter** [comment]
```
OK, great!!
I am testing with MSSQL2000...

I have test to do with 2005 but i saw in previous posts that there are some 
bugs at this stage.

I would be glad to participate to some tests if you need help.
Just let me know.

Regards,
Fabrice

```

Original comment by `fabrice....@gmail.com` on 20 Feb 2012 at 10:41

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in 0.4.4, which I released today. To scan a specific instance, 
enter a system with a backslash and its instance name. For example:

SQLSRV
SQLSRV\instance
SQLSRV\instance2

The first line will scan the default instance. The next two lines will scan 
instances "instance" and "instance2".
```

Original comment by `andrew.O...@gmail.com` on 22 Feb 2012 at 11:12
- Changed state: **Done**
